serverlessworkflow/modules/ROOT/pages/cloud/quarkus/build-workflow-images-with-tekton.adoc

```diff
@@ -1 +1,156 @@
+:keywords: kogito, workflow, serverless, kubernetes, minikube, openshift, containers, tekton, pipeline
+
+This document describes how to build a Workflow Application on a Kubernetes cluster using the Tekton pipeline.
+
+
+.Prerequisites
+You have a git repo containing your workflow and the dockerfile to build the workflow as a docker/podman image
+and a namespace where to install Tekton i.e. `custom-namespace`.
+
+[NOTE]
+====
+
+If you are using Openshift, you can install the Red Hat Pipeline Operator. If you are using Minikube, you can follow this link:https://github.com/tektoncd/pipeline/blob/main/docs/developers/local-setup.md#using-minikube[tekton guide for Minikube].
+If you are using Kind you can follow this link:https://github.com/tektoncd/plumbing/tree/main/hack[Tekton guide for Kind]. If you want to use it on a plain kubernetes cluster, you can follow this link:https://tekton.dev/docs/pipelines/install/[Tekton guide for kubernetes].
+====
+
+
+== Create the pipeline
+In the `custom-namespace` namespace, in the pipelines menu, open the pipeline item, create it, and copy the following yaml file.
+It contains a pipeline to clone your image workflow repository. You can build the image and push it to the local image registry.
+
+----
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: custom-workflow-image-pipeline
+  namespace: custom-namespace
+spec:
+  params:
+    - description: name of the deployment to be patched
+      name: deployment-name
+      type: string
+    - description: url of the git repo for the code of deployment
+      name: git-url
+      type: string
+    - default: pipelines-1.9
+      description: revision to be used from repo of the code for deployment
+      name: git-revision
+      type: string
+    - description: image to be built from the code
+      name: IMAGE
+      type: string
+  tasks:
+    - name: fetch-repository
+      params:
+        - name: url
+          value: $(params.git-url)
+        - name: subdirectory
+          value: ''
+        - name: deleteExisting
+          value: 'true'
+        - name: revision
+          value: $(params.git-revision)
+      taskRef:
+        kind: ClusterTask
+        name: git-clone
+      workspaces:
+        - name: output
+          workspace: shared-workspace
+    - name: build-image
+      params:
+        - name: IMAGE
+          value: $(params.IMAGE)
+        - name: TLSVERIFY
+          value: 'false'
+      runAfter:
+        - fetch-repository
+      taskRef:
+        kind: ClusterTask
+        name: buildah
+      workspaces:
+        - name: source
+          workspace: shared-workspace
+  workspaces:
+    - name: shared-workspace
+----
+
+If you use the Tekton dashboard on Minikube, Kind or on a Kubernets cluster, copy the content of the yam in the pipeline section.
+I you want to skip the UI, save the content of the yaml, with placeholder replaced, in a yaml file called pipeline.yaml and execute
```

Comment:
Which content?
The yaml that you showed before?

Reply:
Yep called custom-workflow-image-pipeline.yaml in the description and in the following sections

Reply:
ok, maybe you could make it more clear do the reader does not gets confused?